Makefile

```diff
@@ -500,9 +504,6 @@ lint: tools doc-lint ## Run linting checks
 	# Linting for the YTT generation test code...
 	cd $(YTT_TESTS_DIR); $(GOLANGCI_LINT) run -v
 
-	# Check licenses in shell scripts and Makefile
-	hack/check-license.sh
```

Comment:
Yeah, this diff is a little confusing I think. It's still there, but now up on (the new) line 493. That keeps it part of the `lint` target since it isn't a `go-lint` specific concern.